src/placeos-driver/interface/electrical_relay.cr

```diff
@@ -0,0 +1,19 @@
+abstract class PlaceOS::Driver; end
+
+module PlaceOS::Driver::Interface::ElectricalRelay
+  abstract def relay(state : Bool, index : Int32 = 0, **options)
```

Comment:
Is encoding an `**options` splat something that's desired from the interface level. If so, we should do this on all methods for consistency.

Reply:
the splat is ignored at the public interface level so can only be used internally in the driver
in this case the desire is for the implementation to pass those options to the `task` so the `pulse` function can manipulate ordering